apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -702,6 +779,28 @@ def grade(element_html, data):
             -1 if num_initial_correct == len(submission) else num_initial_correct
         )
 
+        if feedback_type.startswith("first-wrong"):
+            for i, block in enumerate(student_answer):
+                if i >= num_initial_correct:
```

Comment:
Instead of this break, use a list slice on `student_answer` (like `student_answer[:num_inital_correct]`) in the enumerate call.

Reply:
done